commands/tar.go

```diff
@@ -0,0 +1,60 @@
+package commands
+
+import (
+	"archive/tar"
+	"compress/gzip"
+	"fmt"
+	"io"
+	"os"
+)
+
+// CreateTarball creates a tarball containing all the files in filePaths and writes it to w.
+func CreateTarball(w io.Writer, filePaths []string) error {
```

Comment:
I'm a little reluctant to have these functions in a separate file, because: 

- It seems like it would be better placed in `deploy.go`
- I would expect to see a command for every file in `commands/`, and there isn't a command here

What's the rationale?